oxidation/bindings/generator/api.py

```diff
@@ -25,35 +25,63 @@ class Ref(Generic[TypeVar("REFERENCED")]):  # noqa
 
 # A type that should be converted from/into string
 class StrBasedType:
-    pass
+    ...
```

Comment:
nop, `StrBasedType` should not have an ellipsis: ellipsis is to indicate the body of the class/function is missing where `StrBasedType` is entirely defined here (it is a class marker)

Of course this is only cosmetic given the script doesn't look for the body, but it's more readable to stick with `pass` and only use `...` for binded functions.

Reply:
Done